lib/bosh/BoshDirectorClient.js

```diff
@@ -38,9 +39,9 @@ class BoshDirectorClient extends HttpClient {
   clearCache(config) {
```

Comment:
can this method also be renamed to `clearConfigCache`, since the variable is also named thus?